src/app/pages/adoption/adoption.js

```diff
@@ -20,6 +20,10 @@ function BookSelectorPage({selectedBooksRef}) {
         [selectedBooks]
     );
     const instructions = 'Select titles you are using even if the edition number is different.';
+    const includeFilter = React.useCallback(
+        (b) => !b.comingSoon,
+        []
+    );
```

Comment:
Is this just for performance reasons? Not sure it would do much vs the inline version. Calling `useCallback` has a cost anyway.

Reply:
It's for hygiene reasons. I had the dreaded update-creates-update loop. That might have been due only to the default value, but I covered both bases. The performance difference is negligible.